src/sentry/demo/data_population.py

```diff
@@ -555,82 +636,6 @@ def send_session(self, sid, user_id, dsn, time, release, **kwargs):
         resp = requests.post(url=url, data=body)
         resp.raise_for_status()
 
-    def generate_releases(self, projects):
```

Comment:
@roggenkemper I rather keep this as a class method because at some point I think we might want to store the releases on a the instance so we don't have to query it always

Reply:
moved it back!